_search-plugins/search-pipelines/rerank-processor.md

```diff
@@ -108,11 +124,71 @@ POST /_search?search_pipeline=rerank_pipeline
 ```
 {% include copy-curl.html %}
 
-The `query_context` object contains the following fields. 
+The `query_context` object contains the following fields. You must provide either `query_text` or `query_text_path` but cannot provide both simultaneously.
+
+Field name | Required/Optional | Description
+:--- | :--- | :---  
+`query_text` | Exactly one of `query_text` or `query_text_path` is required. | The natural language text of the question that you want to use to rerank the search results. 
+`query_text_path` | Exactly one of `query_text` or `query_text_path` is required. | The full JSON path to the text of the question that you want to use to rerank the search results. The maximum number of characters allowed in the path is `1000`.
+
+
+<!-- vale off -->
+## The by_field rerank type
+<!-- vale on -->
+Introduced 2.18
+{: .label .label-purple }
+
+To rerank results by a document field, specify the `by_field` rerank type.
+
+The `by_field` object supports the following fields.
+
+Field  | Data type | Required/Optional | Description
+:--- | :---  | :--- | :--- 
+`target_field` | String | Required |  Specifies the field name or a dot path to the field that contains the score by which to rank. 
```

Comment:
"that contains the score by which to rank" => "containing the score to use for ranking/reranking"?